client/src/components/LandingPage/AniCard.js

```diff
@@ -53,20 +53,7 @@ export default function AniCard({ name, text }) {
           </Typography>
         </CardContent>
       </CardActionArea>
-      <CardActions>
-        <Button
-          size="small"
-          className={classes.adjust}
-          style={{
-            backgroundColor: "#f26522",
-            borderRadius: "999999999999999999999999999px"
-          }}
-          color="primary"
-          variant="contained"
-        >
-          Learn More
-        </Button>
-      </CardActions>
+      <CardActions></CardActions>
```

Comment:
If we don't have any action, we can remove this `CardAction`

Reply:
yes Removing